Makefile

```diff
@@ -58,15 +58,16 @@ docs:
 	$(MAKE) -C docs clean
 	$(MAKE) -C docs html
 
-RAIDENVERSION ?= 'master'
+RAIDENVERSION?=master
+GETH_URL_LINUX?='https://gethstore.blob.core.windows.net/builds/geth-linux-amd64-1.7.3-4bb3c89d.tar.gz'
```

Comment:
Thanks @ulope . Any ideas about these definitions here? They're defined in `.travis.yml` but then everyone has to define them locally in order to bundle locally.

Another option would be to not use docker at all, but Konrad wanted that for security reasons (which we'll have on macOS anyways...) and just copy the macOS package creation code.

Reply:
Yeah, I've been thinking about that as well but didn't want to hold up this PR with "unnecessary" bikeshedding. 

But in general I think we need a better way to define those dependencies. IIRC we're up to 4(?) places now that repeat the geth and solc urls. That just cries for some refactoring.

Reply:
guess it's fine to remove these here and having to define them if one wants to bundle locally. do you agree?

Reply:
I guess that's good enough. Bundling locally should happen pretty rarely.